### PR TITLE
Plate effects

### PR DESCRIPTION
**Pull request by mybluecorners**
add better sound effects for plates
plates erupt into glass shards that fly outwards when you smash them on someone
fixed badly balanced throwforce values for shards/knives
removed useless sleep() from plate smash attacks

### CODE REVIEW

File: code/obj/item/kitchen.dm [hunk 1]
```diff
@@ -478,21 +480,33 @@ TRAYS
...
+			for (var/i in 1 to rand(2,3))
+				var/obj/O = unpool(/obj/item/raw_material/shard/glass)
+				O.set_loc(shardturf)
+				if(src.material)
+					O.setMaterial(copyMaterial(src.material))
+				O.throw_at(get_offset_target_turf(shardturf, rand(-4,4), rand(-4,4)), 7, 1)
+
+		qdel(src)
+
+	proc/unique_tap_garbage_fluck(mob/M as mob, mob/user as mob)
```
Comment (flrsh) [inline code/obj/item/kitchen.dm:508]: This looks unintentional.

File: code/obj/item/kitchen.dm [hunk 2]
```diff
@@ -604,6 +618,7 @@ TRAYS
 			unique_attack_garbage_fuck(M, user)
 		else
 			M.visible_message("<span class='alert'>[user] taps [M] over the head with [src].</span>")
+			unique_tap_garbage_fluck(M,user)
```
Comment (flrsh) [inline code/obj/item/kitchen.dm:621]:
```suggestion
			unique_tap_garbage_fuck(M,user)
```

File: code/obj/item/kitchen.dm [hunk 3]
```diff
@@ -758,6 +773,8 @@ TRAYS
 
 		src.visible_message("\The [src] looks less sturdy now.")
 
+	unique_tap_garbage_fluck(mob/M as mob, mob/user as mob)
```
Comment (flrsh) [inline code/obj/item/kitchen.dm:776]: Unintentional?

Comment (mybluecorners) [reply code/obj/item/kitchen.dm:776]: it was intentional, cause its a bad proc name i made an even badder one :)

**Comment by ZeWaka** [comment]
Concern: the glass recycler, this PR will enable you to get infinite glass

File: code/obj/item/kitchen.dm (hunk 3 above)
Comment (flrsh) [reply code/obj/item/kitchen.dm:776]: Mbc why 😿

**Pull request merged**